.github/workflows/mobile-ci.yml

```diff
@@ -0,0 +1,61 @@
+name: Mobile CI
+
+on:
+  push:
+    branches:
+      - feature/nodejs-msa-mobile-*
```

Comment:
Remove branch `feature/nodejs-msa-mobile-*` we are only build CI latest whenever codebase merge to main branch